recipes/pycryptodome/meta.yaml

```diff
@@ -26,6 +26,8 @@ requirements:
 
   run:
     - python
+    - gmp  # [not win]
+    - mpir  # [win]
```

Comment:
You don't need `mpir` here as they are vendoring a copy in windows.

Reply:
`mpir` is in the wheel, but is it also in the source?